provision.sh

```diff
@@ -8,6 +8,13 @@ sudo apt-get update
 sudo apt-get install -y python-pip
 sudo wget -O- https://toolbelt.heroku.com/install-ubuntu.sh | sh
 sudo pip install virtualenvwrapper
+sudo apt-get install -y python-dev libpq-dev postgresql postgresql-contrib
+sudo su - postgres << EOF
```

Comment:
I think you should use `su - vagrant -c ""` syntax.  Check [this](https://github.com/torchbox/vagrant-django-template/blob/master/etc/install/install.sh#L33).

The reason is, the user vagrant would be working. If root owns the DB files, it could cause access problems later.